Source/Scene/Cesium3DTileFeature.js

```diff
@@ -0,0 +1,247 @@
+/*global define*/
+define([
+        '../Core/Color',
+        '../Core/defined',
+        '../Core/defineProperties'
+    ], function(
+        Color,
+        defined,
+        defineProperties) {
+    'use strict';
+
+    /**
+     * Provides access to a feature's properties stored in the tile's batch table, as well
```

Comment:
All of these high-level objects should specifically mention `Cesium3DTileset` as early as possible.  Perhaps a one liner at the top followed by more detailed documentation.